x-pack/plugins/enterprise_search/public/applications/app_search/components/layout/page_template.tsx

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
I don't feel super strongly either way, since this is namespaced (folder-wise... folder-spaced?) to `app_search/` I don't think it's _too_ confusing, but I know there's IDE implications to file-finding or showing tab names 🤷 

FWIW I was kinda copying Workplace Search's folder architecture pattern here (they already have a `layout/nav.tsx` and `layout/kibana_header_actions.tsx` which we also now have our own version of), so I thought `layout/page_template.tsx` made sense as well 🤔 

Definitely doesn't bother me either way though, so happy to revisit later if it's annoying down the road or if we want to circle back and do a vote